polynote-frontend/polynote/ui.js

```diff
@@ -1478,16 +1500,16 @@ export class MainUI extends EventTarget {
         const notebookTab = this.tabUI.getTab(path);
 
         if (!notebookTab) {
-            const notebookUI = new NotebookUI(path, this.socket, this);
+            this.notebookUI = new NotebookUI(path, this.socket, this);
```

Comment:
Thought about doing this in #118, but then you also have to update it when the user changes tabs, which means the tab has to know about the `NotebookUI`... and it just gets kinda hairy. But certainly keeping a reference to the current `NotebookUI` would be a lot cleaner than pulling it out of the `CellsUI` pulled out of the tab's DOM element 😅

Reply:
Oops, didn't check that, yikes!